packages/interledger/codegen.yml

```diff
@@ -8,6 +8,8 @@ generates:
     config:
       defaultMapper: Partial<{T}>
       resolverTypeWrapperSignature: Promise<T>
+      scalars:
+        UInt64: bigint
```

Comment:
bigint isn't an unsigned bigint. Should we keep the scalar as UInt64? 

Also I can't remember but are there any values in the account spec that could be negative?

Reply:
I think the accounts service (and tigerbeetle) only does unsigned values